vendor.conf

```diff
@@ -79,10 +79,10 @@ github.com/golang/protobuf 7a211bcf3bce0e3f1d74f9894916e6f116ae83b4
 # gelf logging driver deps
 github.com/Graylog2/go-gelf v2
 
-github.com/fluent/fluent-logger-golang v1.2.1
+github.com/fluent/fluent-logger-golang 40b03fe3da81939966d25b5e8a6283d08ab2dafa
```

Comment:
ping @tagomoris are there plans to tag a new release? (diff since last release: https://github.com/fluent/fluent-logger-golang/compare/v1.2.1...master)

Reply:
Oh, I missed it. Will do it in this week.

Reply:
Hey @tagomoris , Please ping me once you have added a tag to it so that I can do the changes here.

Reply:
@dungeonmaster18 done!